trade-finance-manager-api/.eslintrc.js

```diff
@@ -35,12 +35,7 @@ const baseRules = {
   ],
   'lines-between-class-members': ['error', 'always', { exceptAfterSingleLine: true }],
   'no-unused-vars': ['error', { ignoreRestSiblings: true }],
-  'object-curly-newline': [
-    'error',
-    {
-      consistent: true,
-    },
-  ],
+  'object-curly-newline': 'off',
```

Comment:
This was contradicting prettier (see https://stackoverflow.com/questions/59318013/eslint-object-curly-newline-conflicts-prettier-multi-line-rationale)